### PR TITLE
Add blog post for week 36

### PR DESCRIPTION
**Pull request by lachmanfrantisek**

### CODE REVIEW

File: weekly/2023/week-36.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+authors: flachman
+tags:
+  - 2023-September
+  - 2023
+  - September
+---
+
+## Week 36 (September 5th – September 11th)
+
+- Packit now supports `commit-message` action that can be used to override the default commit message produced by Packit during `propose-downstream` or `pull-from-upstream`. Please pay attention to our documentation with regards to the usage of this action. ([packit#2070](https://github.com/packit/packit/pull/2070))
```
Comment (lbarcziova) [inline weekly/2023/week-36.md:13]:
```suggestion
- Packit now supports `commit-message` action that can be used to override the default commit message produced by Packit during `propose-downstream` or `pull-from-upstream`. Please pay attention to [our documentation](https://packit.dev/docs/configuration/actions#commit-message-1) with regards to the usage of this action. ([packit#2070](https://github.com/packit/packit/pull/2070))
```

File: weekly/2023/week-36.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- Packit CLI now supports testing the `pull-from-upstream` workflow.
+  Use the `packit pull-from-upstream` command from the `packit` RPM package. ([packit#2063](https://github.com/packit/packit/pull/2063))
+- Packit now passes `initiator` context for tmt to the Testing Farm. You can use this option to run or skip certain tests when they're run by Packit. ([packit-service#2176](https://github.com/packit/packit-service/pull/2176))
```
Comment (lbarcziova) [inline weekly/2023/week-36.md:16]:
```suggestion
- Packit now passes `initiator` context for tmt to the Testing Farm. You can use [this option](https://tmt.readthedocs.io/en/latest/spec/context.html#initiator) to run or skip certain tests when they're run by Packit. ([packit-service#2176](https://github.com/packit/packit-service/pull/2176))
```

File: weekly/2023/week-36.md (hunk 1 above)
Comment (mfocko) [reply weekly/2023/week-36.md:13]:
```suggestion
- Packit now supports `commit-message` action that can be used to override the default commit message produced by Packit during `propose-downstream` or `pull-from-upstream`. Please pay attention to [our documentation](https://packit.dev/docs/configuration/actions#commit-message) with regards to the usage of this action. ([packit#2070](https://github.com/packit/packit/pull/2070))
```

With the `-1` it seems to skip to the env variables

**Review by TomasTomecek — APPROVED** [review]
so much work :O

**Pull request merged**